keps/sig-api-machinery/20190802-dynamic-coordinated-storage-version.md

```diff
@@ -172,6 +172,24 @@ During bootstrap, for each resource, the API server
   to avoid conflicting with other API servers.
 * installs the resource handler.
 
+The above mentioned process requires an API server to update the storageVersion
+before accepting API requests. If we don't enforce this order, data encoded in
+an unexpected version can sneak into etcd. For example, an API server persists a
+write request encoded in an obsoleted version, then it crashes before it can
+update the storageVersion. The storage migrator has no way to detect this write.
+
+For the cmd/kube-apiserver binary, we plan to enforce this order by adding a new
```

Comment:
It seems like any API that isn't related to storage of a resource by the kube-apiserver could still be handled.  This would mean that things like SubjectAccessReview, TokenAccessReview, any read request, and requests to aggregated API servers should be eligible for handling.  Can you more narrowly scope the restriction?

Reply:
Will do.